src/main/scala/org/scalameter/Parameter.scala

```diff
@@ -0,0 +1,15 @@
+package org.scalameter
+
+import org.scalameter.picklers.Pickler
+
+
+case class Parameter[T: Pickler](fullName: String) extends PicklerBasedKey[T] {
+  def pickler: Pickler[T] = implicitly[Pickler[T]]
```

Comment:
Since the `Pickler` argument is implicitly turned into a field anyway (i.e. physically a part of the object state), let's make this explicit - could you make this a `val`?